docs/modules/record-encryption/aws-kms/con-aws-kms-setup.adoc

```diff
@@ -12,17 +12,17 @@ filter to be kept separate from any keys used by other systems. This document as
 will be to prefix the alias used by the filter with the word `KEK_`. If a different naming convention is used, adapt
 the instructions accordingly.
 
-== Administrator Actor
+== Role of the Administrator
 
-To use the filter, there must be an administrative identity established within AWS IAM.  This user, which is likely to be a human,
-has the responsibility to manage keys and aliases within AWS KMS for use by the filter.
+In order to use the filter, there must be an external entity which is responsible for creating encryption keys within AWS KMS which will be used to encrypt the records. It is up to the organization
+deploying Record Encryption to decide how this entity is managed.
```

Comment:
The reason we chose this language was because the "admin" could be a human being, or could be another system.  We are trying to convey "something outside" of Kroxylicious.  Do you have a suggestion?

Reply:
"administrative process" perhaps? that leaves it ambiguous as to whether it's an automated or manual process...

Reply:
how about introducing with "administrator (or administrative process)" and then later references use "administrator". @PaulRMellor @SamBarker WDYT?

Reply:
Yeah I think that works

Reply:
That's great Keith

Reply:
@PaulRMellor pushed up a change, how does that look?

Reply:
Looks good. As mentioned, a few unexpanded comments were mised.